cs/src/Contracts/TunnelListByRegion.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="TunnelListByRegion.cs" company="Microsoft">
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license.
+// </copyright>
+
+using System;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.DevTunnels.Contracts;
+
+/// <summary>
+/// Tunnel list by region.
+/// </summary>
+public  class TunnelListByRegion
+{
+    /// <summary>
+    /// Azure region name.
+    /// </summary>
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? RegionName { get; set; }
```

Comment:
Should we add or reference a list of expected region names to be returned here

Reply:
It'll be one of official Azure region names, as listed by `az account list-locations -o table`.
We can list what current regions are in the comments. That may change later when we onboard new regions.